lib/applicants/resolvers.ts

```diff
@@ -448,7 +447,7 @@ export const verifyIdentity: Resolver<MutationVerifyIdentityArgs, VerifyIdentity
   if (activePermit === null) {
     return {
       ok: false,
-      failureReason: 'APP_DOES_NOT_EXPIRE_WITHIN_30_DAYS',
+      failureReason: 'APP_DOES_NOT_EXPIRE_WITHIN_30_DAYS', // NOTETOSELF: should this be a different reason
       applicantId: null,
     };
   }
```

Comment:
@OustanDing should the failure reason for someone without an active permit also be `APP_DOES_NOT_EXPIRE_WITHIN_30_DAYS`. This would display the following message on the frontend: `Your current permit expiry date is too far away to request a renewal.
      Please check back when your permit is expiring within 30 days or less.` which I feel might be confusing/misleading

Reply:
Resolved, cc @carelynntsai